sdk/core/scheduler/main.cc

```diff
@@ -541,9 +541,9 @@ __cheriot_minimum_stack(0xb0) int futex_timed_wait(Timeout        *timeout,
 	return 0;
 }
 
-__cheriot_minimum_stack(0xc0) int futex_wake(uint32_t *address, uint32_t count)
+__cheriot_minimum_stack(0xd0) int futex_wake(uint32_t *address, uint32_t count)
 {
-	STACK_CHECK(0xc0);
+	STACK_CHECK(0xd0);
```

Comment:
Did you chase down why?

Reply:
No, but it's only one more spill.  Setting sub-object bounds on variable-sized structures can require an extra register.